src/main/java/com/lpvs/auth/SecurityConfig.java

```diff
@@ -33,7 +33,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .and()
                 .logout()
                 .logoutRequestMatcher(new AntPathRequestMatcher("/oauth/logout"))
-                .logoutSuccessUrl("/")
+                .logoutSuccessUrl("http://localhost:3000")
```

Comment:
Maybe we need to move this value to application.properties?

Reply:
Agree. If we run FE on the server, a hardcoded address could be a problem.

Reply:
Moved the value to application.properties.
I'd appreciate it if you could check if the name of the newly added variable is okay.